client/my-sites/earn/memberships/products.jsx

```diff
@@ -388,15 +401,16 @@ class MembershipsProductsSection extends Component {
 						{ translate( 'Custom confirmation message' ) }
 					</h6>
 					<p>
-						{ translate(
-							'Add a custom message to the confirmation email that is sent out for this recurring payment plan.'
-						) }
+						{ translate( 'Add a short message to the confirmation email sent to subscribers.' ) }
 					</p>
-					<FormTextArea
+					<CountedTextarea
 						value={ editedCustomConfirmationMessage }
 						onChange={ ( event ) =>
 							this.setState( { editedCustomConfirmationMessage: event.target.value } )
 						}
+						acceptableLength={ MAX_LENGTH_CUSTOM_CONFIRMATION_EMAIL_MESSAGE }
+						showRemainingCharacters={ true }
+						placeholder={ translate( 'Thank you for subscribing!' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **12** times:
`translate( 'Thank you for subscribing with WordPress.com' )` *ES Score: 7*